packages/pg-v5/commands/credentials/rotate.js

```diff
@@ -16,8 +16,8 @@ async function run(context, heroku) {
     throw new Error('cannot pass both --all and --name')
   }
 
-  if (util.essentialNumPlan(db) || (util.legacyEssentialPlan(db) && cred !== 'default')) {
-    throw new Error("You can't rotate credentials on Essential-tier databases.")
+  if (util.legacyEssentialPlan(db) && cred !== 'default') {
+    throw new Error('You can only rotate default credentials on legacy Essential-tier databases.')
```

Comment:
The new message is correct, but it confused me for a minute.
Maybe "essential-tier databases don't support custom credentials"?

Reply:
Sure, love the help with copy. I'll update the text.